diracx/values.yaml

```diff
@@ -120,7 +120,8 @@ diracx:
     DIRACX_CONFIG_BACKEND_URL: "git+file:///cs_store/initialRepo"
     DIRACX_SERVICE_AUTH_TOKEN_KEY: "file:///signing-key/rsa256.key"
     DIRACX_SERVICE_AUTH_ALLOWED_REDIRECTS: '["http://anything:8000/docs/oauth2-redirect"]'
-
+  configuration:
```

Comment:
I don't think this should be in the `diracx` section

Reply:
Where do you think it should be?